src/xdocs/config_coding.xml

```diff
@@ -3468,6 +3468,22 @@ public class D {
 }
        </source>
        <p>
+         3) If validateOnlyOverlapping is set to false but the variable is locally defined, there
+            will not be a violation provided the variable doesn't overlap, for example:
```

Comment:
We have this new paragraph and then the next paragraph is `Examples of how the check works if validateOnlyOverlapping option is set to false:`.
Both say this is how `validateOnlyOverlapping` for `false` behaves.

It sounds like we are repeating the same text again, and it is not clear how your paragraph is suppose to be different than the next.